syncmaster/db/migrations/versions/2024-09-30_b9f5c4315bb2_.py

```diff
@@ -3,7 +3,7 @@
 """add search_vector with GIN indexed to Transfer table
 
 Revision ID: b9f5c4315bb2
```

Comment:
Shouldn't revision ID be updated to match other ones?

Reply:
means update format of ongoing revision ids?